lib/internal/errors.js

```diff
@@ -727,6 +727,16 @@ const fatalExceptionStackEnhancers = {
   }
 };
 
+// Node uses an AbortError that isn't exactly the same as the DOMException
+// to make usage of the error in userland and readable-stream easier.
+// It is a regular error with `.code` and `.name`.
+class AbortError extends Error {
+  constructor() {
+    super('The operation was aborted');
```

Comment:
Doesn’t error codes usually have ERR as prefix rather than postfix?

Reply:
Yes we discussed this yesterday at the end of the meeting - this is kind of unfortunate.

Unfortunately `AbortError`s on the web already define a `.code` which is `ABORT_ERR` and not `ERR_ABORT` so this is the same `.code` for compatibility sake.

When we weighed "developers will be able to check Node and non/Node errors more easily" with "errors start with ERR_" we figured it was the better tradeoff to make.